Assets/Scripts/Calendar UI.cs

```diff
@@ -80,46 +102,114 @@ private void HandleBackwardClick()
         SetNumberOfBlanks();
         PopulateCalendarGrid();
     }
+    private void HandleCellImageSelect(Cell cell)
+    {
+        foreach (DayCell c in populationList)
+        {
+            if (c != cell)
+            {
+                c.DeSelect();
+            }
+        }
+    }
+    private void HandleSelectCell(Cell cell)
+    {
+        if (pop == null)
+        {
+
+            selectedCell = cell as DayCell;
```

Comment:
should be called selectedDayCell to make it clear this is a concrete low level class and not an abstraction